stories/topics/proc-aap-gcp-application-upgrade.adoc

```diff
@@ -0,0 +1,131 @@
+[id="proc-aap-gcp-application-upgrade"]
+
+Technical Preview
+~~~~~~~~~~~~~~~~~
+
+You can upgrade your existing {AAPonGCP} deployment to the newer version. The upgrade process covers upgrading of automation hub and automation controller. The upgrade process takes roughly the same amount of time as a {AAPonGCP} deployment install. Follow the steps below to ensure a smooth upgrade process.
+
+[NOTE]
+=====
+{AAPonGCP} supports sequential upgrades. All upgrades should be no more than one major version behind what you are currently upgrading to. For example, in order to upgrade to {AAPonGCP} to 2.2.20230215, you must be on the first version of {AAPonGCP}.
+=====
+
+.Prerequisites
+* You require the docker installed in order to run the upgrade playbook.
+
+== Preparing the environment
+
+The upgrade process requires a few volumes to be mounted, so *prepare a fresh directory* to be used for this process.
+
+.Procedure
+. First, create the directories that will be used to configure the upgrade process.
++
+[source,bash]
+----
+# Secrets directory. Store the GCP credentials file.
+$ mkdir secrets
+# Extra vars directory. All the configuration variables for the upgrade process.
+$ mkdir extra_vars
+----
++
+[NOTE]
+=====
+To create the GCP credentials file, see, 
+link:https://developers.google.com/workspace/guides/create-credentials[Create credentials] and 
+link:https://cloud.google.com/iam/docs/keys-create-delete[Create and delete keys]
+=====
++
+. Store your Service account credential json file in secrets dir. 
++
+. Gather the following input variables to create an `extra_vars.yaml` file within `extra_vars` dir.
+.. *gcp_service_account_credentials_json_path*: Service account credential json file path for the google cloud account.
```

Comment:
@leena-jawale: can we give some clarity here?

add something like

*gcp_service_account_credentials_json_path*: Service account credential json file path for the google cloud account. This will be the path within the execution environement container, hence the same path shown here must be used by just changing the appropriate file name you are using.